### DISCUSSION TITLE
Treehole | 去以六月息

### DISCUSSION THREAD

**Discussion by giscus** [ittousei/ittousei.github.io#5] [General]
The discussion is served as a microblog space at the following site:

https://ittousei.github.io/treehole/

**Comment by ittousei** [comment]
Hello, world!

**Comment by Sprinining** [comment]
到此一游~

**Comment by ittousei** [comment]
最近🍑圈终于恢复了，这次得提前留个档，防止再突然挂掉。初步想法是在博客里开个留档栏目。

**Reply by ittousei** [reply]
互联网的记忆周期比想象中要短很多，几年前关注的一些管人里垢，如今不是隐藏就是销号了。以前图方便用截图留档，但照片一旦多了就找不到以前存的图了，做一些系统性的备份还是有必要的。

**Reply by ittousei** [reply]
放在两年前，实现这些想法估计得折腾很久，好在现在是2026年，有codex代写代码了，科技真伟大啊。

**Comment by ittousei** [comment]
5.27更新：
1、🍑⭕️的留档页已上线，详见：https://ittousei.github.io/VupWeiboArchive/
2、在博客中新设了一个文章转载栏目，非原创文章只在Categories中显示，在Home和Archives中隐藏，详见：https://ittousei.github.io/categories/repost/
